npm/http/server/server.go

```diff
@@ -65,12 +65,14 @@ func NewNpmRestServer(listeningAddress string) *NPMRestServer {
 
 func (n *NPMRestServer) GetNpmMgr(npMgr *npm.NetworkPolicyManager) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		npMgr.Lock()
+		// QUESTION(jungukcho) What information does it need from npMgr?
+		// Is this function required to hold lock for npMgr?
```

Comment:
Copying the message from old PR:

This function is used by Dataplane parser Quang uses, to get the whole internal cache for PodMap, Nsmap and netpolmap, since this is a read operation, i will let your design drive if lock is required.

But please do make sure that this function will return the 3 maps along with listmap and setmap

Reply:
Thank you for copying it. 

Sure. After reading Quang's PR, I realize it relies on NPM cache information.
I will see first how nicely expose NPM cache information to external components to minimize costs at controller side (e.g., lock cost)

Reply:
agreed, this is an optimization we can do later, at this moment i am okay with a primitive approach.